src/artemis/nexus_writing/write_nexus.py

```diff
@@ -231,7 +232,10 @@ def __enter__(self):
 
     def __exit__(self, *_):
```

Comment:
**should**: Can you add a docstring on the `__exit__` which says why we're creating the temp file etc?

Reply:
Done